docusaurus/docs/Android/04-compose/07-guides/06-implementing-own-capabilities.mdx

```diff
@@ -0,0 +1,233 @@
+        onShowMoreReactionsSelected: () -> Unit,
+        modifier: Modifier = Modifier,
+        shape: Shape = ChatTheme.shapes.bottomSheet,
+        overlayColor: Color = ChatTheme.colors.overlay,
+        reactionTypes: Map<String, ReactionIcon> = ChatTheme.reactionIconFactory.createReactionIcons(),
+        @DrawableRes showMoreReactionsIcon: Int = R.drawable.stream_compose_ic_more,
+        onDismiss: () -> Unit = {},
+        headerContent: @Composable ColumnScope.() -> Unit = {
+            // Default content, uses own capabilities to regulate sending reactions
+        },
+        centerContent: @Composable ColumnScope.() -> Unit = {
+            // Default content
+        },
+)
+```
+
+The file containing `SelectedReactionsMenu` can be
+found [here](https://github.com/GetStream/stream-chat-android/blob/0f1f54e912baf7a7284d9a4d1435fb6743aa9c78/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/selectedmessage/SelectedReactionsMenu.kt#L62)
+
+`SelectedReactionsMenu` menu regulates the following capability:
+
+- send-reaction
+
+### SelectedChannelMenu
+
+```kotlin
+@Composable
+public fun SelectedChannelMenu(
+        selectedChannel: Channel,
+        isMuted: Boolean,
+        currentUser: User?,
+        onChannelOptionClick: (ChannelAction) -> Unit,
+        onDismiss: () -> Unit,
+        modifier: Modifier = Modifier,
+        shape: Shape = ChatTheme.shapes.bottomSheet,
+        overlayColor: Color = ChatTheme.colors.overlay,
+        headerContent: @Composable ColumnScope.() -> Unit = {
+            // Default content
+        },
+        centerContent: @Composable ColumnScope.() -> Unit = {
+            // Default content, uses own capabilities to regulate
+            // deleting and leaving channels
+        },
+)
+```
+
+`SelectedChannelMenu` contains a `Channel` value. It uses the `ownCapabilities` property contained by that `Channel`
+value to leverage own capabilities inside the default center content.
+
+The file containing `SelectedChannelMenu` can be
+found [here](https://github.com/GetStream/stream-chat-android/blob/0f1f54e912baf7a7284d9a4d1435fb6743aa9c78/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/info/SelectedChannelMenu.kt#L66)
+
+`SelectedChannelMenu` regulates the following capabilities:
+
+- leave-channel
+- delete-channel
+
+## Other own capabilities
+
+There are channel capabilities which are not implemented in the SDK because we do not offer the corresponding components
+or the functionality.
+You can implement these yourself after creating the correct components.
+
+These are as follows:
+
+- freeze-channel
+- set-channel-cooldown
+- search-message
+- update-channel
+- update-channel-members
+- ban-channel-members
+
+Additionally, the capability 'send-typing-events' is implemented in our `MessageComposerController` used by
+our `MessageComposerViewModel`.
+Please keep this in mind should you wish to not use our `ViewModel`s.
```

Comment:
Similarly to the previous file - I would show a single example here, e.g. the MessageComposer (as the selected message menu is not connected to the message list and it would be extra work) and how it behaves without capabilities and with some.